src/main_thread/decrypt/types.ts

```diff
@@ -122,6 +122,8 @@ export interface IProtectionData {
   content?: IContent;
   /** Every initialization data for that type. */
   values: IInitDataValue[];
+
+  forceSessionRecreation?: boolean | undefined;
```

Comment:
Wouldn't this one be more at its place in `IProcessedProtectionData`, as in: it's a property added by the ContentDecryptor internals.

`IProtectionData` is relied on by external code to provide to us new protection initialization data and I fail to see for now what they would do with that.

Reply:
Done